ui/src/app/components/resource-card.tsx

```diff
@@ -162,14 +162,18 @@ export class ResourceCard extends React.Component<Props, State> {
   }
 
   get actionsDisabled(): boolean {
-    return !this.writePermission;
+    return !this.writePermission && !this.isConceptSet;
   }
 
   get writePermission(): boolean {
     return this.props.resourceCard.permission === 'OWNER'
       || this.props.resourceCard.permission === 'WRITER';
   }
 
+  get deletePermission(): boolean {
```

Comment:
Nit: I'd probably name this `ownPermission` or something to match above, or change `writePermission` to `editPermission` (or something similar). Just for semantic consistency. I lean towards `own/ownerPermission` because nothing stops us from adding other owner only actions.

Reply:
👍